language/test/Language_test.rb

```diff
@@ -162,6 +164,86 @@
     end
   end
 
+  describe "#Save" do
+    let(:language) { "es_ES" }
+    let(:initial_stage) { false }
+    let(:command_result) { { "exit" => 0 } }
+
+    before do
+        allow(Yast::Stage).to receive(:initial).and_return(initial_stage)
+
+        allow(Yast::WFM).to receive(:Execute).and_return(command_result)
+        allow(Yast::SCR).to receive(:Execute).and_return(command_result)
+        allow(Yast::SCR).to receive(:Write)
+        allow(subject).to receive(:valid_language?).with(language).and_return(true)
+
+        subject.Set(language)
+        subject.SetDefault
+    end
+
+    it "writes .sysconfig.language.INSTALLED_LANGUAGES" do
```

Comment:
this is a bit low level to my taste. What actually you do it that you modify /etc/sysconfig/language value INSTALLED_LANGUAGES. Ideally tested with switched SCR on real file, that is really written down, but it is just nice to have. But better description what method do is almost must have.